micropub.php

```diff
@@ -917,17 +940,24 @@ public static function store_geodata( $args ) {
 				}
 				$args['meta_input']['geo_latitude']  = $props['latitude'][0];
 				$args['meta_input']['geo_longitude'] = $props['longitude'][0];
+				$args['meta_input']['geo_altitude']  = $props['altitude'][0];
 			} elseif ( 'geo:' === substr( $location, 0, 4 ) ) {
 				// Geo URI format:
 				// http://en.wikipedia.org/wiki/Geo_URI#Example
-				// https://indiewebcamp.com/micropub##location
+				// https://indieweb.org/micropub##location
```

Comment:
the wiki doesn't support fragmentions, so this link doesn't work. let's use https://indieweb.org/Micropub#h-entry instead.